### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by adrianhajdin**

### CODE REVIEW

File: client/components/LinkListPage/Links/Links.jsx [hunk 1]
```diff
@@ -20,9 +23,9 @@ const link = (props) => {
 };
 
 link.propTypes = {
-  url: PropTypes.string,
-  title: PropTypes.string,
-  tags: PropTypes.string,
+  url: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+  tags: PropTypes.isRequired,
```
Comment (Rejam) [inline client/components/LinkListPage/Links/Links.jsx]: Tags could perhaps be optional. They might not be required for every resource.

File: client/components/Menu/Menu.jsx [hunk 2]
```diff
@@ -36,51 +20,45 @@ const styles = theme => ({
...
 });
 
 class ResponsiveDrawer extends React.Component {
-  state = {
-    mobileOpen: false,
-  };
+  constructor(props) {
+    super(props);
+    this.state = { mobileOpen: false };
+  }
```
Comment (Rejam) [inline client/components/Menu/Menu.jsx]: I think this was why DevLee wanted the class-prop plugin, can set state without using the constructor.

File: client/components/Menu/Menu.jsx [hunk 3]
```diff
@@ -91,11 +69,11 @@ class ResponsiveDrawer extends React.Component {
           {['Forum', 'Creators', 'Contact', 'Setting'].map((text, index) => (
```
Comment (Rejam) [inline client/components/Menu/Menu.jsx]: Don't think this line needs index either does it?

File: client/components/Menu/Menu.jsx [hunk 4]
```diff
@@ -91,11 +69,11 @@ class ResponsiveDrawer extends React.Component {
...
 
-            {/* Original code */}
-            {/* <ListItemIcon>{index % 2 === 0 ? <InboxIcon /> : <MailIcon />}</ListItemIcon> */}
-            {/* Need to figure out how to correctly add icon */}
+              {/* Original code */}
+              {/* <ListItemIcon>{index % 2 === 0 ? <InboxIcon /> : <MailIcon />}</ListItemIcon> */}
+              {/* Need to figure out how to correctly add icon */}
 
-              <ListItemIcon>{index % 2 === 0 ? <Add/> : <Face />}</ListItemIcon>
+              <ListItemIcon>{index % 2 === 0 ? <Add /> : <Face />}</ListItemIcon>
```
Comment (Rejam) [inline client/components/Menu/Menu.jsx]: index % 2 === 0 is the same as !(index % 2)
Or in other words:
`index % 2 ? <Face /> : <Add />`

File: client/components/Menu/Menu.jsx [hunk 5]
```diff
@@ -157,11 +129,11 @@ class ResponsiveDrawer extends React.Component {
         <main className={classes.content}>
           <div className={classes.toolbar} />
           <Typography paragraph>
-          <h1>Welcome to the Programming Reference Site</h1>
+            <h1>Welcome to the Programming Reference Site</h1>
           </Typography>
           <Typography paragraph>
           This project was created to create a website to help individuals programmers, ranging from novice to advanced, to have a reference site, where specific links, githubs, articles, etc. are listed and recommended by programmers such as yourself!
-          We understand the importance of using a search engine, but we also understand that there are so many resources out there, sometimes you don't know what is good or not! 
+          We understand the importance of using a search engine, but we also understand that there are so many resources out there, sometimes you don‘t know what is good or not!
```
Comment (Rejam) [inline client/components/Menu/Menu.jsx]: Heh, this apostrophe looks rotated 180deg to me

File: client/components/Menu/Menu.jsx (hunk 4 above)
Comment (aselee) [reply client/components/Menu/Menu.jsx]: Right! This was based on the basic response drawer code with using material-ui icons. I was messing with it a little bit, but I think that code will have to be changed essentially.

File: client/components/Menu/Menu.jsx (hunk 3 above)
Comment (aselee) [reply client/components/Menu/Menu.jsx]: No, it shouldn't! The index parameter was used for the matching the material-ui icons. Should've taken that off, my bad!

File: client/components/LinkListPage/Links/Links.jsx (hunk 1 above)
Comment (aselee) [reply client/components/LinkListPage/Links/Links.jsx]: Noted!

Comment (WarrenHellman) [reply client/components/LinkListPage/Links/Links.jsx]: The way I was imagining it, each link will need at least 1 tags (language) and possibly more like level (beginner intermediate advanced)

**Pull request merged**